src/overlays/actors/ovl_kaleido_scope/z_kaleido_scope_PAL.c

```diff
@@ -167,21 +167,149 @@ static u16 D_8082ABEC[] = {
     PAUSE_MAP, PAUSE_EQUIP, PAUSE_QUEST, PAUSE_ITEM, PAUSE_EQUIP, PAUSE_MAP, PAUSE_ITEM, PAUSE_QUEST,
 };
 
+// Controls if the item slot specifided is able to be equipped and the color of the name card
 u8 gSlotAgeReqs[] = {
-    1, 9, 9, 0, 0, 9, 1, 9, 9, 0, 0, 9, 1, 9, 1, 0, 0, 9, 9, 9, 9, 9, 0, 1,
+    AGE_REQ_CHILD, // Deku stick
+    AGE_REQ_NONE,  // Deku Nuts
+    AGE_REQ_NONE,  // Bombs,
+    AGE_REQ_ADULT, // Bow
+    AGE_REQ_ADULT, // Fire arrows
+    AGE_REQ_NONE,  // Dins fire,
+    AGE_REQ_CHILD, // Slingshot
+    AGE_REQ_NONE,  // Ocarina,
+    AGE_REQ_NONE,  // Bombchus
+    AGE_REQ_ADULT, // Hookshot
+    AGE_REQ_ADULT, // Ice arrows
+    AGE_REQ_NONE,  // Farore's Wind
+    AGE_REQ_CHILD, // Boomerang
+    AGE_REQ_NONE,  // Lens of truth
+    AGE_REQ_CHILD, // Bean
+    AGE_REQ_ADULT, // Megaton Hammer
+    AGE_REQ_ADULT, // Light arrows
+    AGE_REQ_NONE,  // Nayrus love
+    AGE_REQ_NONE,  // Bottle 1
+    AGE_REQ_NONE,  // Bottle 2,
+    AGE_REQ_NONE,  // Bottle 3,
+    AGE_REQ_NONE,  // Bottle 4,
+    AGE_REQ_ADULT, // Adult trade
+    AGE_REQ_CHILD  // Child trade
 };
 
 u8 gEquipAgeReqs[][4] = {
-    { 0, 1, 0, 0 },
-    { 9, 1, 9, 0 },
-    { 0, 9, 0, 0 },
-    { 9, 9, 0, 0 },
-};
-
-u8 gItemAgeReqs[] = {
-    1, 9, 9, 0, 0, 9, 1, 9, 9, 9, 0, 0, 0, 9, 1, 9, 1, 0, 0, 9, 9, 9, 9, 9, 9, 9, 9, 9, 9,
-    9, 9, 9, 9, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
-    0, 1, 0, 0, 1, 9, 0, 9, 0, 0, 9, 0, 0, 1, 1, 1, 0, 0, 0, 9, 9, 9, 1, 0, 0, 9, 9, 0,
+    {
+        AGE_REQ_ADULT, // Quiver/Bullet bag (Unused)
+        AGE_REQ_CHILD, // Kokiri sword
+        AGE_REQ_ADULT, // Master Sword
+        AGE_REQ_ADULT  // Big Goron's Sword
+    },
+    {
+        AGE_REQ_NONE,  // Bomb bag
+        AGE_REQ_CHILD, // Deku shield
+        AGE_REQ_NONE,  // Hylian shield
+        AGE_REQ_ADULT  // Mirror shield
+    },
+    {
+        AGE_REQ_ADULT, // Strength (Unused)
```

Comment:
why put `(Unused)` here but not on `Bomb bag` and `Scale`? I guess because this has `AGE_REQ_ADULT` but I would still add `(Unused)` to the other two for clarity (they are unused too right)